pkg/manifest/manifest.go

```diff
@@ -187,15 +187,32 @@ func resolveRelativeLinks(node *Node, _ *Node, manifest *Node, r registry.Interf
 	return nil
 }
 
-func extractFilesFromNode(node *Node, parent *Node, manifest *Node, r registry.Interface) error {
+func checkFileTypeFormats(node *Node, _ *Node, manifest *Node, r registry.Interface, ContentFileFormats []string) error {
+	if node.Type != "file" {
+		return nil
+	}
+	file := node.FileType.File
```

Comment:
Let's check all the fields from FileType(`Source` and `MultiSource`). Also we can use https://pkg.go.dev/slices#example-ContainsFunc for simplicity